### ISSUE TITLE
Incorrect path for pidfile 

### ISSUE THREAD

**Issue by xasima** [softek/puppet-zabbixagent#1]
I have almost clean ubuntu 12.04 system managed remotely by puppet-agent.
With the initial configuration I have noticed the following problem in the log file 

```
zabbix_agentd [23172]: cannot create PID file 
[/var/run/zabbix/zabbix_agentd.pid]: [2] No such file or directory
```

There is no /var/run/zabbix, but  /var/run/zabbix-agent folder were created instead. 
So, when perform this change in /etc/puppet/modules/zabbixagent/templates/zabbix_agentd.conf.unix.erb

```
+ PidFile=/var/run/zabbix-agent/zabbix_agentd.pid
- PidFile=/var/run/zabbix/zabbix_agentd.pid //Deleted
```

everything goes fine. 

Just for information purposes I present  my current puppet declaration (on puppet-master)

```
node 'metrics-fe1' {
 # - Ensure correctness of APT configuration -------------
 class { 'apt':
    proxy_port  => '3128',
    proxy_host  => 'proxy.metrics',
 }
 # - Add ppa with latest zabbix-agent
 apt::ppa { "ppa:h.tims/upstream-experimental": }

 # - Install all required packages --------------
 $base_packages = ['ntp'] # tried  ['ntp', zabbix-agent'] for the first time...,
 # ... but confict with zabbixagent below with message ' Duplicate declaration: Package[zabbix-agent] is already declared in file /etc/puppet/manifests/site.pp at line XX; cannot redeclare on node'

 package { $base_packages:
    ensure => installed,
 }
 class { 'zabbixagent':
    servers => 'zabbix-master.metrics',
 }
}
```

**Comment by smerchek** [comment]
At first glance, this looks like an issue caused by differences between the 1.8.x and 2.0.x versions of Zabbix. The configuration file template that I used was generated by the 1.8.x package of Zabbix. A solution would be to make the PidFile parameter configurable; however, there may be some other changes to the config file in 2.0 that should be considered. 

I'd love for this puppet module to support Zabbix 2.x, so I'll look into this in the next couple of days.

@xasima do you know of any other differences in the configuration/installation of Zabbix 2.x?

**Comment by smerchek** [comment]
It seems there are a lot of differences between the config file for 1.8.x and 2.x. 

I think the best approach will be to use the [inifile puppet module](http://forge.puppetlabs.com/cprice404/inifile) to only modify the select settings that we need, leaving the rest of the file unmodified. This should insulate us from most other changes that could happen in the config file from version to version.

**Comment by smerchek** [comment]
Fixed by b0ae27bf0d4f36fbc2f99e273190314b3895d792
Available in version [0.1.0](http://forge.puppetlabs.com/softek/zabbixagent/0.1.0)
